### DISCUSSION TITLE
`ctx-watch` with print formatting

### DISCUSSION THREAD

**Discussion by TornaxO7** [pwndbg/pwndbg#1420] [Q&A] [answered]
Hello! I'm trying to add a `ctx-watch` which prints a variable in hexa.

I've tried the following:
```
ctx-watch "p /x" 10
```
but `pwndbg` still prints it out as `10` and not `0xa`. May I ask what I'm doing wrong?

**Comment by gsingh93** [comment, marked as answer]
Can you try `ctx-watch execute "p/x 10"`?

**Reply by TornaxO7** [reply]
nice, it's working! Thank you!
